### ISSUE TITLE
AttributeError: module 'autopy' has no attribute 'mouse

### ISSUE THREAD

**Issue by x011** [autopilot-rs/autopy#60]
Traceback (most recent call last):

``` 
File "/home/user/script.py", line 366, in <module>
    autopy.mouse.move(500, 550)
AttributeError: module 'autopy' has no attribute 'mouse
```
ubuntu 20

Any idea how to fix this?

**Comment by msanders** [comment]
This may be a duplicate of #58. Is there another folder in your current working directory named "autopy"? If so, can you change directories and try again?

**Comment by x011** [comment]
No, I don't have file named `autopy` on the same dir, but uninstalling the version installed by `root` and re-installing using `-U`  seems to solve the problem:

`pip3 uninstall -y autopy
pip3 install autopy -U`
